tajo-plan/src/main/java/org/apache/tajo/plan/LogicalPlanner.java

```diff
@@ -1963,7 +1964,7 @@ public static boolean checkIfBeEvaluatedAtGroupBy(EvalNode evalNode, GroupbyNode
   }
 
   public static boolean checkIfBeEvaluatedAtJoin(QueryBlock block, EvalNode evalNode, JoinNode node,
-                                                 boolean isTopMostJoin) {
+                                                 boolean isOnPredicate) {
```

Comment:
```isOnPredicate``` seems to be a flag to indicate if a predicate is ON clause?

But, it seems to be inconsistent with its usecases.  In ProjectionPushDownRule, TRUE is given to this parameter. But, in the line 1153, this value is true even though they are not placed ON clause.

Could you let me know your exact intention about this change?